contributing/releasing.md

```diff
@@ -62,6 +62,12 @@ Note: if for some reason `cut` fails, first ensure that nobody else is in the mi
 You will now have a local `release-candidate` branch, a new section in CHANGELOG.md titled
 "release-candidate", and the `release-candidate` branch will have been pushed to GitHub.
 
+#### Create the PR fore the PR
```

Comment:
Nit/spelling: s/fore/for and s/the PR/the release branch/

Suggestion: "Create a Pull Request for the Release Branch"